src/stores/loyaltyPass/LoyaltyPassStore.tsx

```diff
@@ -14,22 +13,6 @@ export const useLoyaltyPassStore = createWithEqualityFn(
       pdas: [],
       timestamp: undefined,
 
-      setValue: (key: keyof SettingsProps, value: any) =>
```

Comment:
are we sure it is never used?

Reply:
Yes, likely we don´t use it. Otherwise, we create our own store-setters.